src/components/include/DefaultImportModalDialog.jsx

```diff
@@ -0,0 +1,320 @@
+import nativeFormats from "@exabyte-io/made.js/lib/parsers/native_formats";
```

Comment:
let's call this `nativeFormatParsers`

Reply:
And import Made directly

Reply:
renamed to nativeFormatParsers (and in made.js native_format_parsers.js respectively), will import Made directly